projects/project-player-library/src/lib/pages/task-details-page/task-details-page.component.html

```diff
@@ -13,7 +13,7 @@ <h4 class="break-all">{{task.name}}</h4>
         <h3>{{"END_DATE"|translate}}</h3>
         <div>
             <mat-form-field class="w-full">
-                <input  matInput [matDatepicker]="picker" [(ngModel)]="task.endDate" (ngModelChange)="onDateChange($event)" disabled>
+                <input  matInput [matDatepicker]="picker" [(ngModel)]="task.endDate" (ngModelChange)="onDateChange($event)" [min]="minDate" [max]="maxDate" disabled>
```

Comment:
Can u modify this so that if there is project startDate or endDate it takes that value else use the minDate and maxDate which have been declared